portfolio/src/main/webapp/index.html

```diff
@@ -7,7 +7,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body onload="getName()">
+  <body onload="getName(); getLogin()">
```

Comment:
I would make a single JS function that calls both of these and call that new function here.